CMakeLists.txt

```diff
@@ -6,6 +6,9 @@
 cmake_minimum_required(VERSION 3.1)
 project(mariadbclientpp)
 
+option(MARIADBPP_TEST "Run mariadbpp tests" ON)
```

Comment:
Please change this to "Build mariadbpp tests" as they won't be run automatically.

Reply:
You are right, it makes more sense.